doc/Getting-started.md

```diff
@@ -36,7 +38,35 @@ cd my-first-admin
 npm install ng-admin --save
 ```
 
-**Tip**: For now, you will be using a version of ng-admin that packages all dependencies into a single file, including angular.js itself. When you need to package ng-admin manually with other dependencies, you can use the standalone version (see [Getting Ready For Production](Production.md)).
+The `ng-admin.min.js` file is a ready-to-use version of ng-admin, packaging all dependencies required to run the application, including Angular.js.
+
+### With a Module Bundler
+
+If you use a module bundler such as Webpack, you can use `ng-admin` using a simple `require`:
+
+``` js
+require('ng-admin');
+```
+
+Including `ng-admin` this way would also ng-admin SASS files to generate styles. Hence, you would need
```

Comment:
Remove this paragraph and add the requirement of either the SASS or CSS version in previous snippet.